web/src/compose.js

```diff
@@ -324,7 +324,7 @@ export function finish(from_do_schedule_message = false) {
 
     compose_ui.show_compose_spinner();
 
-    if (!compose_validate.validate()) {
+    if (!compose_validate.validate(from_do_schedule_message)) {
```

Comment:
`scheduling_message` is probably a clearer variable name -- the point is what we're trying to do, not which function happens to be the way we do that.